src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -92,7 +92,7 @@ public override ExportResult Export(in Batch<Metric> batch)
 
                     var metricType = metric.MetricType;
 
-                    if (metricType.IsHistogram())
+                    if (metricType == MetricType.Histogram || metricType == MetricType.ExponentialHistogram)
```

Comment:
won't IsHistogram() cover both this already?

Reply:
No not today. I'm not so certain how useful these extension methods on MetricType will be for us.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/094124e9ee61466a5bbf182d923658cde2c2dcf5/src/OpenTelemetry/Metrics/MetricTypeExtensions.cs#L62-L65

https://github.com/open-telemetry/opentelemetry-dotnet/blob/094124e9ee61466a5bbf182d923658cde2c2dcf5/src/OpenTelemetry/Metrics/MetricTypeExtensions.cs#L30

https://github.com/open-telemetry/opentelemetry-dotnet/blob/094124e9ee61466a5bbf182d923658cde2c2dcf5/src/OpenTelemetry/Metrics/MetricType.cs#L66-L74